add process questions and deming cycle (#26)
epiverse-trace/research-compendium: episodes/introduction.Rmd

- Keep track of changes, and 
- Manuscript writing.

> But before incorporating any new practice, first, we can evaluate the processes we are following. A process-centred approach helps us to remove the focus on human error and be aware that processes can fail people with good intentions, and we can enter a cycle of continuous improvement.

Author: ```suggestion
But before incorporating any new practice, first, we can evaluate the **processes** we are following. A process-centred approach helps us remove the focus on human error, be aware that processes can fail people with good intentions, and accept that we can enter a continuous improvement cycle.
```

to assume (aside from nefarious actors) that the analyst did not want that error
to occur. Given that she thought she was producing an analysis free from errors,
you must look at the way she developed the analysis to understand where the
error occurred, and create safeguards so that the error does not occur again." ([Parker, 2017a](https://sachsmc.github.io/rpackage-workshop/opinionated-analysis-dev.pdf) and [Parker, 2017b](https://posit.co/resources/videos/opinionated-analysis-development/))

Repetitive events (like outbreak response and research data analysis projects) give us the opportunity to:

- Focus on the **process** we have followed,
- Evaluate **where** bottlenecks occur, and then
- **Adopt** new practices to be better protected against errors in the next iteration.

::::::::::::::::: callout

### Deming Cycle

This approach aims to follow a [Deming cycle](https://www.lean.org/lexicon-terms/pdca/) of Plan, Do, Check, and Act, as a foundation for continuous improvement.

:::::::::::::::::::::::::

::::::::::::::::: discussion

Exercise: Your experience analyzing outbreak data (the latest... or the most chaotic!)

Take 5 minutes.

Reflect on these questions:

> - How do you organize your files and folders?

Author: ```suggestion
- How do you **organize** your files and folders?
```

> - Where does it describe what your project did or how to use it? Was it all in one place?

Author: ```suggestion
- Where do you **describe** what your project does or how to use it? Was it all in one accessible place?
```

> - Could your project be reused by colleagues? Do you think it is?

Author: ```suggestion
- Could your project be **reused** by colleagues? Do you think it is?
```
